docs/src/tutorials/tutorial_4.md

```diff
@@ -0,0 +1,304 @@
+# **Tutorial 4:** Do you know our Mascot? - Learn about transitions and subactions!
+
+You have learned a couple of cool features of Javis already. Now you're ready to finally meet our little mascot. Well actually you can't just see him, we have to create him first. :smile:
```

Comment:
You'll need to manually add in the emoji. I generally use the julia repl and do `clipboard("\:smile:")` and that works.

Reply:
thanks